src/main/resources/modules/cystic_fibrosis.json

```diff
@@ -0,0 +1,1649 @@
+        }
+      ],
+      "range": {
+        "low": 1,
+        "high": 2,
+        "unit": "days"
+      }
+    },
+    "Oxygen_Therapy": {
+      "type": "Procedure",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 65710008,
+          "display": "Oxygen Therapy"
+        }
+      ],
+      "duration": {
+        "low": 30,
+        "high": 60,
+        "unit": "minutes"
+      },
+      "conditional_transition": [
+        {
+          "transition": "death_3",
+          "condition": {
+            "condition_type": "PriorState",
+            "name": "patient_dies"
+          }
+        },
+        {
+          "transition": "Release_From_Hospital"
+        }
+      ]
+    },
+    "Release_From_Hospital": {
+      "type": "EncounterEnd",
+      "direct_transition": "Life_With_Cystic_Fibrosis"
+    },
+    "patient_lives": {
+      "type": "Simple",
+      "conditional_transition": [
+        {
+          "transition": "Sepsis_Of_Paeruginosa",
+          "condition": {
+            "condition_type": "PriorState",
+            "name": "Develop_Infection_1"
+          }
+        },
+        {
+          "transition": "Sepsis_of_Staphylococcus_aureus",
+          "condition": {
+            "condition_type": "PriorState",
+            "name": "Develop_Infection_2"
+          }
+        },
+        {
+          "transition": "pulmonary_exacerbation"
+        }
+      ]
+    },
+    "pulmonary_exacerbation": {
+      "type": "ConditionOnset",
+      "assign_to_attribute": "CF_Complication",
+      "target_encounter": "Emergency_Room",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 65710008,
+          "display": "Acute respiratory failure (disorder)"
+        }
+      ],
+      "direct_transition": "wheezing"
+    },
+    "Diabetes_Guard": {
+      "type": "Simple",
+      "conditional_transition": [
+        {
+          "transition": "diseaseSupplier",
+          "condition": {
+            "condition_type": "PriorState",
+            "name": "Develop_Diabetes"
+          }
+        },
+        {
+          "transition": "Develop_Diabetes"
+        }
+      ]
+    },
+    "no_double_infection": {
+      "type": "Simple",
+      "conditional_transition": [
+        {
+          "transition": "Quarterly_Checkup",
+          "condition": {
+            "condition_type": "PriorState",
+            "name": "Develop_Infection_1"
+          }
+        },
+        {
+          "transition": "Develop_Infection_1"
+        }
+      ]
+    },
+    "no_double_infection_2": {
+      "type": "Simple",
+      "conditional_transition": [
+        {
+          "transition": "Quarterly_Checkup",
+          "condition": {
+            "condition_type": "PriorState",
+            "name": "Develop_Infection_2"
+          }
+        },
+        {
+          "transition": "Develop_Infection_2"
+        }
+      ]
+    },
+    "Lung_Transplant_Wait": {
```

Comment:
Right now this lung transplant encounter lasts 4-5 hours. There probably should be an inpatient hospital stay here that lasts at least several days.